app/brave_generated_resources.grd

```diff
@@ -303,14 +303,24 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         Tor
       </message>
 
-      <!-- Sync Infobar -->
+      <!-- Sync Infobars -->
       <message name="IDS_BRAVE_SYNC_V2_MIGRATE_INFOBAR_MESSAGE" desc="Message text shown to users who used a previous version of Brave Sync and need to perform setup again for the new version.">
         Brave Sync has been upgraded and requires setup
       </message>
       <message name="IDS_BRAVE_SYNC_V2_MIGRATE_INFOBAR_COMMAND" desc="Command text shown to users who used a previous version of Brave Sync and need to perform setup again for the new version.">
         Begin Setup
       </message>
 
+      <message name="IDS_BRAVE_SYNC_CANNOT_RUN_INFOBAR_MESSAGE" desc="Message text shown to users when Sync service cannot run and additional attention is required">
+        Sync service could not be started, Sync is not running
```

Comment:
Should this say "Brave Sync is not running" since other infobar refers to it as "Brave Sync"

Reply:
you are right, thank you

fixed at https://github.com/brave/brave-core/compare/9f94be62f077007af6b124c089e3dfc5b6efd6b7..980e5dc37026a0bd8bb1e898e35f88e51ce1f5c2#diff-6314ab9f5910c0e3dd53f8ac6bf09b584020abe6790abe7ebcdacc5756c5d5f9R315